apps/mobile/src/hooks/useBalances.test.ts

```diff
@@ -0,0 +1,53 @@
+import { renderHook, waitFor } from '@/src/tests/test-utils'
+import { useBalances } from './useBalances'
+import { http, HttpResponse } from 'msw'
+import { server } from '@/src/tests/server'
+import { GATEWAY_URL } from '@/src/config/constants'
+
+// Mock active safe selector using a mutable variable
+let mockActiveSafe: { chainId: string; address: string } | null = {
+  chainId: '1',
+  address: '0x123',
```

Comment:
Nit: for a more realistic scenario consider using a valid address. E.g. with faker: 
```suggestion
  address: faker.finance.ethereumAddress(),
```